src/source/feeds.rst

```diff
@@ -31,6 +82,701 @@ XML file.
+====== ====================================================
+
+The provided (desired) status may differ from the resulting one, depending on the other conditions.
+For instance, budget may be depleted, or the seller may have too many active ads already in the category.
+
+======= =============================
+Example .. code-block:: html
+        
+         <ad:status>PAUSED</ad:status>
+======= =============================
+
+.. index:: url
+.. _feed_url:
+
+url
+"""
+
+An external URL that is shown on the ad page. This must be a valid http(s)
+url.
+
+======= =======================================
+Example .. code-block:: html
+        
+         <ad:url>https://www.bmw.de</ad:url>
+======= =======================================
+
+.. index:: vanityUrl
+.. _feed_vanityUrl:
+
+vanityUrl
+"""""""""
+
+The text/url that will be displayed instead of the url in :ref:`feed_url`.
+
+======= ==================================
+Example .. code-block:: html
+        
+         <ad:vanityUrl>BMW</ad:vanityUrl>
+======= ==================================
+
+.. index:: priceType
+.. _feed_priceType:
+
+priceType
+"""""""""
+
+Must be a valid price type identifier from the list of :ref:`price_types`.
+
+======= ============================================
+Example .. code-block:: html
+        
+         <ad:priceType>FIXED_PRICE</ad:priceType>
+======= ============================================
+
+.. index:: price
+.. _feed_price:
+
+price
+"""""
+
+The meaning of the value depends on the :ref:`feed_priceType`. 
+
+If it is `FIXED_PRICE` or `BIDDING_FROM` then **price** is mandatory and needs to be greater than 0.
+The maximum allowed **price** value is ``10000000000`` of the local market currency. (100.000.000,00 EUR / CAD / ... ).
+
+======= =============================
+Example .. code-block:: html
+        
+         <ad:price>1500</ad:price>
+======= =============================
+
+.. index:: originalPrice
+.. _feed_originalPrice:
+
+originalPrice
+""""""""""""""
+
+Product price before discount. Ignored if a seller does not have a discount option enabled. Must be greater than :ref:`feed_price`.
+The maximum allowed **originalPrice** value is ``10000000000`` of the local market currency. (100.000.000,00 EUR / CAD / ... ).
+
+======= ============================================
+Example .. code-block:: html
+        
+         <ad:originalPrice>1500</ad:originalPrice>
+======= ============================================
+
+.. index:: media
+.. _feed_media:
+
+media
+"""""
+
+Complex type used currently for product images. **<media>** should contain from 0 to N **<image>** ordered elements, where the exact limit depends on the category in taxonomy. **<image>** elements must contain a complete URL link pointing to an image on a publicly available webserver.
+
+Allowed image formats: JPEG, JPG, PNG, GIF\*, BMP.
+
+\* Please note that GIFs are not recommended format as thay are only 256 colors or less.
```

Comment:
Animated gifs are also not supported :)

Reply:
https://github.es.ecg.tools/ecg-icas/icas/pull/6822/files#r2006388